packages/ckeditor5-ckbox/tests/ckboxediting.js

```diff
@@ -232,13 +237,16 @@ describe( 'CKBoxEditing', () => {
 				theme: 'default',
 				tokenUrl: 'http://cs.example.com'
 			} );
+			expect( console.warn.callCount ).to.equal( 1 );
```

Comment:
It would be nice, to check the warning text as well. I.e. if it contains `ckbox-config-not-found`.